packages/quantic/cypress/integration/recent-queries-list/recent-queries-list.cypress.ts

```diff
@@ -0,0 +1,100 @@
+import {configure, reset} from '../../page-objects/configurator';
+import {InterceptAliases, interceptSearch} from '../../page-objects/search';
+import {RecentQueriesListExpectations as Expect} from './recent-queries-list-expectations';
+import {scope} from '../../reporters/detailed-collector';
+import {
+  clearInput,
+  performSearch,
+  setQuery,
+} from '../../page-objects/actions/action-perform-search';
+
+interface RecentQueriesListOptions {
+  maxLength: number;
+  label: string;
+  isCollapsed: boolean;
+}
+
+function createRandomQueriesList(length: number) {
```

Comment:
I want to generate a big number of queries to test `maxLength` , and I don't like create big predefined list of string 😆

Reply:
You could also use a smaller value for `maxLength` :wink: . I personally try to avoid random values in tests to ensure that all test runs are testing the exact same thing. But it is mostly a matter of preference, I guess. If you prefer to generate queries, I'm ok with that.